tools/console_version

```diff
@@ -0,0 +1 @@
+COMMIT="0f6df0caf415cbc8ea61260783c86e63baed7416"
```

Comment:
Obviously we have to pin a version of the console rather than pulling the latest `main`, but it's interesting because the console is itself built against a particular version of the API. Usually there's some flexibility in what versions go together because the API doesn't change _that_ much, but this won't really feel good until we're on some kind of semantic versioning scheme and can say "ok, console is for API 1.23.0, which means it should run ok against API 1.23.2"

Reply:
Yeah, I agree with that long-term inclination - this `tools` directory is emulating `Cargo.toml` (by specifying the git commit, instead of semver) but also emulating `Cargo.lock` (by storing the precise SHA256 of the artifact).

Long-term, I think it would be nice to specify API versions, but be able to be able to build/run against specific commits that satisfy those bounds.